src/player.html

```diff
@@ -25,13 +25,16 @@
 		<script src="player.js"></script>
 	</head>
 	<body ng-cloak>
+		<div id="ariaLivePolite" aria-live="polite" aria-atomic="true"></div>
+		<div id="ariaLiveAssertive" aria-live="aggressive" aria-atomic="true"></div>
```

Comment:
Your assertive live region has an `aria-live` value or `aggressive`, so it's not actually being considered a live region.